### PR TITLE
update cbrs layer to use tileserver

### PR DESCRIPTION
**Pull request by tyler-whitman**

### CODE REVIEW

**Review by kent-williams — APPROVED** [review]
LGTM!

File: components/Map/Layers/HexCoverageLayer.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const HOTSPOT_COLOR = '#29d391'
 const DATA_COLOR = '#58a7f9'
 const DC_THRESHOLD = 100
 
-const TILESERVER_URL =
+export const TILESERVER_URL =
```
Comment (Perronef5) [inline components/Map/Layers/HexCoverageLayer.js:10]: Should the tile server url not be placed together with our other urls in some útil file ?

File: components/Map/Layers/CbrsLayer.js [hunk 2]
```diff
@@ -1,95 +1,90 @@
...
-  const { selectedHex } = useSelectedHex()
-
-  const cbrsData = useMemo(() => {
-    try {
-      return require('../../../data/cbrs.csv')
-    } catch (e) {
-      return null
-    }
-  }, [])
+const CELL_HOTSPOT_COLOR = '#D23E72'
```
Comment (Perronef5) [inline components/Map/Layers/CbrsLayer.js:12]: Could this color not be pulled from somewhere else or is it specific to this component?

**Review by Perronef5 — COMMENTED** [review]
Looks good just left a few comments.

File: components/Map/Layers/HexCoverageLayer.js (hunk 1 above)
Comment (tyler-whitman) [reply components/Map/Layers/HexCoverageLayer.js:10]: This one is specific to the MapBox layer, I think leaving it in this coverage layer is fine for the time being. I was first trying to integrate the 5g layer into this one, but the way mapbox works with these layer it doesnt let you change props on the fly it needs a copy of each layer to be loaded to work. So I ended up doing it the way it is where we have a seperate file with a new layer specifically for 5g but it needed a couple things from this file.

File: components/Map/Layers/CbrsLayer.js (hunk 2 above)
Comment (tyler-whitman) [reply components/Map/Layers/CbrsLayer.js:12]: Its specific to this component, the color of the 5g hotspots on the map

File: components/Map/Layers/HexCoverageLayer.js (hunk 1 above)
Comment (Perronef5) [reply components/Map/Layers/HexCoverageLayer.js:10]: Gotcha sounds good

**Review by Perronef5 — APPROVED** [review]
LGTM! 5G layer looks so cool!

**Pull request merged**